examples/language-sdk-instrumentation/rust/README.md

```diff
@@ -1,2 +1,148 @@
 ## Continuous Profiling for Rust applications
+
 ### Profiling a Rust Rideshare App with Pyroscope
+
+![Image](https://github.com/user-attachments/assets/9a6e50a1-b8df-4923-9632-79ace3fea216)
+
+Note: For documentation on Pyroscope's Rust integration visit our website
+for [rust push mode](https://grafana.com/docs/pyroscope/latest/configure-client/language-sdks/rust/)
+
+## Background
+
+In this example we show a simplified, basic use case of Pyroscope. We simulate a "ride share" company which has three
```

Comment:
Removing first person (we), which is a best practice recommended in the [Writers' Toolkit](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#address-users-clearly).
```suggestion
This example shows a simplified, basic use case of Pyroscope that uses a "ride share" company which has three
```